README.md

```diff
@@ -188,6 +188,15 @@ This option allows you to select a particular page from a draw.io file to
 export. Note that an invalid page-index will revert to one of the other valid
 pages (draw.io binary functionality).
 
+### Page Name
+- *Formal Name*: `:page-name:`
+- *Default Value*: `None`
```

Comment:
I'm not sure if we should raise an exception if the page-name is invalid.

I see there is an issue: https://github.com/modelmat/sphinxcontrib-drawio/issues/16. I think it's better to raise an exception to inform the user about this.

Reply:
> You should, but as far as I know it's not possible (per the linked issue).

The page_name_to_index method has parsed the draw.io file, so we can raise DrawIOError if we failed to find the page-name. 
For page-index, it can also be judged by the result of parsing. But I'm not sure if the format of draw.io XML will be modified later, even if the XML file format is modified, a forward-compatible format should be provided.

> 
> (I suspect also it might be possible to get them to add page-name as a possible argument to the draw.io binary - I'm happy to include this but I would prefer it being upstream who parses the draw.io file)

Yes, it will be better if the upstream support it.

Reply:
Yes， we shouldn't allow this too. I can raise an exception for this.

Reply:
> Yes, I think an error would be acceptable.
> 
> Also the default value is not _literally_ `:page-index:`, it's fallback. I was thinking perhaps having both `page-index` & `page-name` should be disallowed? I don't think it really makes sense to allow both

The last version has made this an exception.